scripts/run-external.sh

```diff
@@ -6,15 +6,49 @@ set -o pipefail
 # error on unset variables
 set -u
 
-if [[ -z "${1:-}" ]]; then
-	echo "ERROR: missing cluster name"
-	echo "usage: ❯ $0 <clustername>"
-	exit 1
-fi
-
-apiserver=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$1\")].cluster.server}")
-cafile=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$1\")].cluster.certificate-authority}")
-certfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$1\")].user.client-certificate}")
-keyfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$1\")].user.client-key}")
-
-./operator --apiserver="${apiserver}" --ca-file="${cafile}" --cert-file="${certfile}" --key-file="${keyfile}"
+run() {
+	echo " ❯ $* "
+	"$@"
+}
+
+update_crds() {
+	kubectl replace -f example/prometheus-operator-crd-full ||
+		kubectl create -f example/prometheus-operator-crd-full
+
+	echo "Wait for CRDs to be registered"
+	kubectl wait --for=condition=Established crds --all --timeout=120s
+}
+
+run_operator() {
+
+	local apiserver cafile certfile keyfile
+
+	apiserver=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$cluster\")].cluster.server}")
+	cafile=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$cluster\")].cluster.certificate-authority}")
+	certfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$cluster\")].user.client-certificate}")
+	keyfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$cluster\")].user.client-key}")
+
+	run ./operator \
+		--apiserver="${apiserver}" \
+		--ca-file="${cafile}" \
+		--cert-file="${certfile}" \
+		--key-file="${keyfile}"
+
+}
+
+main() {
+	cd $(git rev-parse --show-toplevel)
+
+	local cluster="${1:-}"
+	if [[ -z "$cluster" ]]; then
+		echo "ERROR: missing cluster name"
+		echo "usage: ❯ $0 <clustername>"
+		exit 1
+	fi
+
+	run make operator
```

Comment:
I was also thinking of adding `ensure_po_non_installed` to check that PO isn't running in cluster already.

Reply:
That seems like a good idea, it it's not too complicated

Reply:
I'm not sure how you could check this.

Reply:
Heuristics, ... is what I had in mind .. but it may be easier to `kubectl delete -f bundle.yaml` anyway.

Reply:
I think it may be sufficient to warn user if a deployment already exists and an option to override that check.